### PR TITLE
Added bio to larsenm and added to index.html

### PR DESCRIPTION
**Pull request by massimolarsen**

### CODE REVIEW

File: users/larsenm/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Current Research
+
+---
+
+### Quasi Monte Carlo
+
+My current research focuses on the implementation of Quasi Monte Carlo methods, for use in radiography calculations, into full scale production codes at LANL. QMC differs from normal MC in it's use of non-pseudorandom number generators. Instead, QMC employs a low-discrepancy sequence which helps to significantly reduce the convergence rate when compared to it's random counterpart.
+
+I am working on two different approaches to implement into LANL's radiography code. Both of the methods will be implemented on soley the source parameters while leaving the rest of the Monte Carlo code unhcanged. The first method is a very standard implementation of the halton sequence which provides the exact low-discrepancy number generation that we need. The second method is slightly more out of the ordinary and uses a python package of the VEGAS algorithm (https://vegas.readthedocs.io/en/latest/index.html) to produce numbers that are biased similarly to something like importance sampling.
```
Comment (aaronjamesreynolds) [inline users/larsenm/index.md]: Solely and unchanged are misspelled.

**Review by aaronjamesreynolds — COMMENTED** [review]
Caught a couple typos.

File: users/larsenm/index.md (hunk 1 above)
Comment (massimolarsen) [reply users/larsenm/index.md]: Fixed

Comment (aaronjamesreynolds) [reply users/larsenm/index.md]: Looks good to me. Once @brancokm or @clemekay give it the okay we can merge.

File: users/larsenm/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+I received my B.S. in Nuclear Engineering from the University of California, Berkeley in 2020.
+
+## Current Research
+
+---
+
+### Quasi Monte Carlo
+
+My current research focuses on the implementation of Quasi Monte Carlo methods, for use in radiography calculations, into full scale production codes at LANL. QMC differs from normal MC in it's use of non-pseudorandom number generators. Instead, QMC employs a low-discrepancy sequence which helps to significantly reduce the convergence rate when compared to it's random counterpart.
```
Comment (clemekay) [inline users/larsenm/index.md]: (you want its here not it's but like again nobody but me cares)

**Review by clemekay — APPROVED** [review]

**Comment by clemekay** [comment]
Fixed same merge conflict as Eric's

**Comment by brancokm** [comment]
Looks good, Merging in.

**Pull request merged**